recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/recipes-graphics/images/core-image-weston.bbappend

```diff
@@ -5,6 +5,11 @@ IMAGE_INSTALL_append = " \
     xen-xenstat \
 "
 
+# Add Pulseaudio to build
```

Comment:
please add the below to the IMAGE_INSTALL_append above

Reply:
Done